### DISCUSSION TITLE
Bootstrap 5 - twinkling tooltip when is the SVG icon.

### DISCUSSION THREAD

**Discussion by Martussky** [twbs/bootstrap#31910] [General]
Hi, first of all, I saw this discussion about the issue with the SVG icon within the tooltip: https://github.com/twbs/bootstrap/discussions/31618. And it looks a little better than before, but still, as we can see it in your documentation here: https://v5.getbootstrap.com/docs/5.0/components/tooltips/#examples when we hover on tooltip background and after on the icon and then again on the background I see that tooltip is twinkling. Will you fix this?
Best, Marta

**Comment by ascensus-mdb** [comment]
I noticed the same problem. Works fine in v4.

Is fixing it already in the backlog?
